docs/source/cli.rst

```diff
@@ -0,0 +1,118 @@
+
+.. _cli:
+
+Command Line Interface
+=======================================
+
+.. highlight:: console
```

Comment:
Hmm not sure if easily configurable... but the rendered code blocks are pretty pale:

![image](https://user-images.githubusercontent.com/3383210/131723612-35fb0b18-aef8-44a2-a57c-139f782e8243.png)

Reply:
Funny enough, during some other formatting earlier today I changed this to use code blocks instead of :: and now it's better.
![Screenshot 2021-09-02 at 12 31 44](https://user-images.githubusercontent.com/13608913/131828689-07b8ca23-ae94-4588-8cfd-2bef16ea63e7.png)